beanie/odm/utils/init.py

```diff
@@ -574,6 +574,9 @@ async def init_document(self, cls: Type[Document]) -> Optional[Output]:
         build_info = await self.database.command({"buildInfo": 1})
         mongo_version = build_info["version"]
         cls._database_major_version = int(mongo_version.split(".")[0])
+
+        own_settings = cls.__dict__.get("Settings")
+        class_id_value = getattr(own_settings, "class_id_value", None)
```

Comment:
```suggestion
        document_settings = cls.get_settings()
        class_id_value = getattr(document_settings, "class_id_value", None)
```
Please reuse code that already exists instead of accessing the raw `__dict__` directly.